components/button/stories/template.js

```diff
@@ -158,13 +160,15 @@ export const TextOverflowTemplate = (args) => Container({
       customStyles: {
         "max-inline-size": "480px",
       },
+      isLabelWrapped: true,
```

Comment:
```suggestion
```
It looks like this was an older variable naming?

Reply:
Yes I need to get rid of this. This was from a previous approach of making the button label wrap by default.